go/vt/vtgate/planbuilder/testdata/aggr_cases.txt

```diff
@@ -2243,7 +2243,7 @@ Gen4 plan same as above
 
 # syntax error detected by planbuilder
 "select count(distinct *) from user"
-"syntax error: count(distinct *)"
+"syntax error at position 24"
 Gen4 plan same as above
```

Comment:
nit: this can be moved to parse_test file now, as planner does not  throw syntax error for functions.